### PR TITLE
Make those extensions work with Jupyter version 51fabeba530b1d50fa3255f48e97d24d72e605d7.

### PR DESCRIPTION
**Pull request by Konubinix**
Jupyter requires the modules to be correctly defined and to provide the
load_ipython_extension variable.

To be honest, I don't understand how it works. I just now that it works with a fresh install of Jupyter from the source code.

### CODE REVIEW

**Comment by juhasch** [comment]
Thank you for your effort, this is welcome. Could you split the changes into smaller chunks (i.e. one for each extension).
I find it difficult to look at these changes en block, and I suspect a few of the extension need a little additional work to bring them back to life, too.

**Comment by Konubinix** [comment]
Thank you for looking at it.

To be honest, I don't know what I did since I don't have now time to dig
into Jupyter source code. I just made sure the extensions I wanted
worked well by fixing the errors my Chromium inspector complained
about. To do so, I only mimicked the behaviour of working extensions.

Of course I can provide smaller patches. I will probably do that during
next week.

**Comment by damianavila** [comment]
Sound like a good idea... I can probably provide some feedback when the thing is splitted... thanks @Konubinix

**Comment by juhasch** [comment]
Needs rebasing after big reorganization PR #273

File: publishing/printview/main.js [hunk 1]
```diff
@@ -6,34 +6,42 @@ define([
...
+
     /**
      * Call nbconvert using the current notebook server profile
      *
      */
 	var nbconvertPrintView = function () {
-		var kernel = IPython.notebook.kernel;
 		var name = IPython.notebook.notebook_name;
-		var command = 'ip=get_ipython(); import os; os.system(\"ipython nbconvert --profile=%s --to html '
+		var command = 'ip=get_ipython(); import os; os.system(\"jupyter nbconvert --profile=%s --to html '
```
Comment (JanSchulz) [inline publishing/printview/main.js]: AFAIK, jupyter has no concept of "profiles"?!

Comment (juhasch) [inline publishing/printview/main.js]: Yes, I already fixed this here but wanted to do the reorganization first.

**Comment by juhasch** [comment]
Closing as outdated

**Pull request closed without merging**